TramsDataApi/Gateways/ConcernsTeamCaseworkGateway.cs

```diff
@@ -34,11 +33,9 @@ public async Task<ConcernsCaseworkTeam> GetByOwnerId(string ownerId, Cancellatio
                 throw new ArgumentNullException(nameof(ownerId));
             }
 
-            var team = await _tramsDbContext.ConcernsTeamCaseworkTeam
+            return await _tramsDbContext.ConcernsTeamCaseworkTeam
```

Comment:
Do you need to get the team members again? I need to pull the branch and check the code again, but it looks like you're just returning the tracked entity, which I guess wouldn't hit the database so no harm in it...

Reply:
Ignore that - it looked like it was part of Add.... #facepalm